packages/shared/src/components/buttons/GetExtensionButton.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { BrowserGroupIcon } from '../icons/Browsers';
+import { IconSize } from '../Icon';
+
+type GetExtensionButtonProps = {
+  className?: string;
+};
+
+export const GetExtensionButton = ({
+  className,
+}: GetExtensionButtonProps): ReactElement => {
+  return (
+    <a
```

Comment:
I think it might still be better to use the existing Button component, then just add the `!text-white` to fix the issue.

Reply:
It's not just the text-white, there's a bunch of class overrides, and quite a few importants (height, font, padding, etc), at which point I think it's better to just make a separate component 😛

Reply:
I guess it's non guidelined the whole thing.
I did think we had it somewhere, but can't find it now maybe it got removed before.

Reply:
I see. In that case, we can maybe extract this component and name it `InstallExtensionButton` - just so we won't have to create another one again.